Sources/FormattingHelpers.swift

```diff
@@ -1890,6 +1891,8 @@ extension Formatter {
                 return "Overridden Functions"
             case .swiftUIProperty, .swiftUIMethod:
                 return "Content"
+            case .swiftUIDynamicProperty:
+                return "SwiftUI Dynamic Properties"
```

Comment:
Is "dymamic property" a common term of art here? I'm not sure I've seen it before. Instead, could we just call these SwiftUI properties?

```suggestion
            case .swiftUIProperty:
                return "SwiftUI Properties"
```

Reply:
There is an existing case `swiftUIProperty` so I renamed this to `swiftUIPropertyWrapper`